tests/integration/test_ext_hms.py

```diff
@@ -123,3 +131,44 @@ def test_migration_job_ext_hms(ws, installation_ctx, prepare_tables_for_migratio
             assert "spark.sql.hive.metastore.version" in job_cluster.new_cluster.spark_conf
         if ws.config.is_aws:
             assert "spark.databricks.hive.metastore.glueCatalog.enabled" in job_cluster.new_cluster.spark_conf
+
+
+@retried(on=[NotFound, InvalidParameterValue], timeout=timedelta(minutes=5))
+def test_running_real_assessment_job_ext_hms(
+    ws, installation_ctx, env_or_skip, make_cluster_policy, make_cluster_policy_permissions
+):
+    # this test spins up clusters using ext hms cluster policy, which will have a startup time of ~ 7-10m
+    # skip this test if not in nightly test job or debug mode
+    if os.path.basename(sys.argv[0]) not in {"_jb_pytest_runner.py", "testlauncher.py"}:
+        env_or_skip("TEST_NIGHTLY")
+
+    ext_hms_ctx = installation_ctx.replace(
+        skip_dashboards=False,
```

Comment:
```suggestion
        skip_dashboards=True,
```
if you want to skip dashboards, then have it as true and then remove the `skip_tasks` hack